src/Opdex.Platform.Domain/Models/Markets/MarketSummary.cs

```diff
@@ -40,6 +41,7 @@ public MarketSummary(ulong marketId, ulong createdBlock) : base(createdBlock, cr
     public decimal DailyStakingUsdChangePercent { get; private set; }
     public decimal ProviderRewardsDailyUsd { get; private set; }
     public decimal MarketRewardsDailyUsd { get; private set; }
+    public ulong LiquidityPoolCount { get; }
```

Comment:
Should this be `uint` to match the other areas?

Reply:
Yes, updated